### PR TITLE
[FIX] sale_order_line_description: missing translation for field description_sale

### PR DESCRIPTION
**Pull request by tafaRU**

### CODE REVIEW

File: sale_order_line_description/models/sale.py [hunk 1]
```diff
@@ -30,12 +30,12 @@ def product_id_change(
...
             uos=False, name='', partner_id=False, lang=False,
             update_tax=True, date_order=False, packaging=False,
-            fiscal_position=False, flag=False
+            fiscal_position=False, flag=False, context=None
     ):
         res = super(SaleOrderLine, self).product_id_change(
             pricelist, product, qty, uom, qty_uos, uos, name,
             partner_id, lang, update_tax, date_order, packaging,
-            fiscal_position, flag)
+            fiscal_position, flag, context)
```
Comment (pedrobaeza) [inline sale_order_line_description/models/sale.py]: Take the occasion to name each keyword argument in case any inheritance change the argument order

Comment (tafaRU) [inline sale_order_line_description/models/sale.py]: Ok.

This change has no sense with new API, I'm still working on it.

Comment (pedrobaeza) [inline sale_order_line_description/models/sale.py]: Of course, I only see the diff and don't look at the @api.multi...

**Comment by tafaRU** [comment]
@pedrobaeza could you please have a look at it?

**Comment by eLBati** [comment]
LGTM :+1:

**Comment by pedrobaeza** [comment]
:+1:

**Comment by guewen** [comment]
:+1:

**Pull request merged**